website/docs/docs/dbt-cloud-apis/metadata-querying.md

```diff
@@ -34,6 +34,11 @@ Every query will rely on a _jobID_.  You can get the jobID by clicking into the
 There are several illustrative example queries in this documentation. You can see an example of [queries on the Model node](/docs/dbt-cloud-apis/metadata-schema-model).
 
 
+## Reasonable use
+All Metadata (GraphQL) API requests are subject to request rate and response size limiting to prevent abuse and help ensure performance and stability. 
+
+The current rate limit is 200 requests within a minute for a given IP address. If a user exceeds this limit, they will receive a HTTP 429 response status.
```

Comment:
What does response size limiting look like for us? You mention it, but unlike the request rate limit there's no context on how that might manifest or how to be a responsible API consumer. 

For an example, GH's GraphQL API limits the number of nodes queryable to 500k: https://docs.github.com/en/graphql/overview/resource-limitations#node-limit

Reply:
just curious - is there a warning for users if they get close to this limit?

Reply:
@joellabes great point, we added response size limit details. we haven't implemented this yet, but plan to soon. the node limit will likely change, but want to establish expectations for usage best practices

@mirnawong1 good idea, but at least for now upon hitting the limit, the user will just receive the error and need to retry. we calibrated the rate limit to the far upper end of API usage